.github/workflows/generate-desktop-targets.yml

```diff
@@ -7,32 +7,38 @@ on:
     paths:
       # The workflow can be triggered by modifying FLEET_DESKTOP_VERSION env.
       - '.github/workflows/generate-desktop-targets.yml'
```

Comment:
Given we are removing FLEET_DESKTOP_VERSION, we can remove the whole `push:` part.

Reply:
Still need push I think, but I can remove the `branch` and `paths` parts

Reply:
Ah, yes, only push tags 👍